org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/views/RuleDescriptionWebView.java

```diff
@@ -148,17 +150,26 @@ private void showRuleDescription(IMarker element) {
       return;
     }
 
-    var issuable = Adapters.adapt(element.getResource(), ISonarLintIssuable.class);
-    var project = issuable.getProject();
-
-    var resolvedBindingOpt = SonarLintCorePlugin.getServersManager().resolveBinding(project);
-    if (resolvedBindingOpt.isPresent()) {
-      new AsyncDisplayRuleDescriptionJob(project, resolvedBindingOpt.get(), ruleKey, browser).schedule();
-    } else {
-      var ruleDetails = SonarLintCorePlugin.getInstance().getDefaultSonarLintClientFacade().getRuleDescription(ruleKey);
-      browser.updateRule(ruleDetails);
+    try {
+      var project = Adapters.adapt(element.getResource(), ISonarLintIssuable.class).getProject();
+      var resolvedBindingOpt = SonarLintCorePlugin.getServersManager().resolveBinding(project);
+      if (resolvedBindingOpt.isPresent()) {
+        var configurationScopeId = resolvedBindingOpt.get().getProjectBinding().projectKey();
```

Comment:
Configuration scope ID is not the projectKey, but the "identifier" of the Eclipse project. The backend is already aware of the binding, so it will deal with that on its own.
I suggest to create a method in the `SonarLintBackendService` to encapsulate the conversion ISonarLintProject to configScopeId.